src/refactor/sat_cli.py

```diff
@@ -0,0 +1,17 @@
+from click import option
+from groundStation import groundStation
+from options import optionsFactory
+
+class sat_cli(groundStation):
+    def run(self):
+        while(1):
+            inStr = self.inputHandler.getInput("$ ")
+            commandStr = "{}.cli.send_cmd({},{})".format(self.satellite, len(inStr), inStr)
```

Comment:
Are sat_cli command's formatted differently to regular commands? From looking at the test scripts, arguments were separated by spaces (ie. `ex2.housekeeping.get_hk(1 0 0)`) instead of commas. Is this detail important or does the input parser handle this case?

Reply:
sat_cli commands are typed like you would in the terminal, as in 
$ ls
I decided to ditch space delimiters entirely and only support comma delimiters. It just makes more sense

Reply:
Oh, I was talking about the commandStr format (eg. `ex2.cli.send_cmd(ls,2)`) which is comma delimited. With this refactor, are all commands with multiple args now comma delimeted? (eg. `ex2.housekeeping.get_hk(1 0 0)` turns to `ex2.housekeeping.get_hk(1,0,0)`)

Reply:
All commands are now comma-delimited (side effect: sat_cli commands can't have commas in them). No more space delimiters on any command